src/KubernetesClient/Enums/WatchEventType.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Runtime.Serialization;
+
+namespace k8s.Enums
+{
+    /// <summary>Describes the type of a watch event.</summary>
+    public enum WatchEventType
```

Comment:
why do you want to move ns?
this is a breaking change

Reply:
I can not move namespaces, but it seems (and perhaps I'm wrong here) that we should break up classes so they don't all live in the root of the source?